app/helpers/patients_helper.rb

```diff
@@ -73,4 +73,11 @@ def dcaf_pledge_limit_help_text
 
     "Pledge Limit Guidelines:<br />1st trimester (7-12 weeks): $#{first_tri}<br />2nd trimester (12-24 weeks): $#{second_tri}<br />Later care (25+ weeks): $#{later_care}"
   end
+
+  def duration_since_last_call
+    @patient = Patient.find(params[:id])
+    days_since_last_call = (Time.now - @patient.calls.last.created_at).to_i / 86400
+    if days_since_last_call => 120, return more_than_120
```

Comment:
We can just return `days_since_last_call` here, I don't think we need to overthink it and we can use this in a bunch of places.